Packs/Core/Playbooks/playbook-Office_process_creates_a_scheduled_task_via_file_access_README.md

```diff
@@ -0,0 +1,73 @@
+This playbook handles "Office process creates a scheduled task via file access" alerts.
+
+Playbook Stages:
+
+Analysis:
+During the analysis, the playbook will perform the following:
+
+- Checks the Office file path for any suspicious locations.
+- Checks the initiator process prevalence.
+- Checks the initiator process reputation.
+- Extracts the local path of the Office file and runs a script to calculate the Office file hash.
+- Checks if the hash of the Office file identified as malicious.
+- Checks if the initiator process is non-prevalent with suspicious reputation or path.
+
+Investigation:
+During the alert investigation, the playbook will perform the following:
+
+- Searches for related Cortex XSIAM alerts and insights on the endpoint by specific alert names or by the following MITRE techniques to identify malicious activity: T1055 - Process Injection, T1566 - Phishing.
+
+Remediation:
```

Comment:
can we add the conditions for the remediation actions? for example, in case of related alerts found.....automatically disable the malicious scheduled task